### DISCUSSION TITLE
Customising FormLabel's RequiredIndicator in custom theme?

### DISCUSSION THREAD

**Discussion by mjgwood** [chakra-ui/chakra-ui#4131] [Q&A] [answered]
It's possible to pass a custom required indicator component (the red asterisk) to any single FormLabel using the `indicator` prop e.g. `<FormLabel indicator={CustomIndicator}>`, but is it possible to simply customise the indicator colour for ALL indicators in my custom theme?

I've had no issue customising other components' styles (such as FormLabel's default colours), but I've not been able to figure out how to customise RequiredIndicator. I know I could create a custom FormLabel component that uses the `indicator` prop but I'd prefer to keep all my customisations in the theme if possible.

Here are the references to RequiredIndicator:

https://github.com/chakra-ui/chakra-ui/blob/main/packages/form-control/src/form-label.tsx
https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/form.ts

Any help would be much appreciated.

**Comment by vcastroi** [comment, marked as answer]
check docs, [customize-theme > customizing-component-styles](https://chakra-ui.com/docs/theming/customize-theme#customizing-component-styles)

here is how to change the RequiredIndicator's color to green
```
const theme = extendTheme({ components: { Form: { baseStyle: { requiredIndicator: { color: 'green' } } } } });
```

**Reply by mjgwood** [reply]
Thanks very much for answering! I couldn't find a solution in the docs for my issue.

I didn't realise I had to reference the Form component, since I didn't do that for other components e.g. FormLabel. I think I was trying to treat RequiredIndicator as a 'top-level' component, but now I better understand what's going on [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/form.ts).

Thanks again!
